spec/wings/value_mapper_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+require 'spec_helper'
+require 'wings/value_mapper'
+
+RSpec.describe Wings::ValueMapper do
+  subject(:mapper) { described_class.for(value) }
+  let(:value)      { 'a value' }
+  let(:uri)        { RDF::URI('http://example.com/moomin') }
+
+  describe '.for' do
+    it 'returns a value mapper' do
+      expect(described_class.for(value)).to be_a described_class
```

Comment:
Do you think it makes sense to shorten this to `expect(mapper)`?

Reply:
i like keeping the `.for` call explicit here, since it's what we're testing. `mapper` could change in the future to support other tests.